schemas/1.6.0/adaptive-card.json

```diff
@@ -303,6 +303,11 @@
           "description": "Specifies the background image of the card.",
           "version": "1.2"
         },
+        "metadata": {
+          "$ref": "#/definitions/Metadata",
+          "description": "Defines various metadata properties typically not used for rendering the card",
+          "version": "1.6"
+        },
```

Comment:
I am not sure how this ended up in this diff - this was added and merged in #7105

Reply:
@paulcam206 is this an issue?

Reply:
I'll take a look :)

Reply:
Great, resolving this convo